### ISSUE TITLE
Git ignored files not clear

### ISSUE THREAD

**Issue by chazmcgrill** [alexdauenhauer/catppuccin-noctis#33]
Thanks for the great theme.

Its hard to see difference between gitignored and normal files in vscode (appears to be the same color)

https://github.com/alexdauenhauer/catppuccin-noctis/blob/4ce8ba6b275f5bd558ed7f0c16f9c30ba7661adf/themes/Catppuccin%20Noctis%20Mocha-color-theme.json#L237C53-L237C60

Using the Mocha theme

Am currently overriding using this:
```
    "workbench.colorCustomizations": {
        "gitDecoration.ignoredResourceForeground": "#757FAA"
    }
```

**Comment by alexdauenhauer** [comment]
thanks for flagging this! great suggestion, I'll get to it soon!

**Comment by alexdauenhauer** [comment]
this is fixed in 2.1.1! https://github.com/alexdauenhauer/catppuccin-noctis/releases/tag/2.1.1

**Comment by chazmcgrill** [comment]
Many thanks
